internal/pkg/deploy/cloudformation/stack/env.go

```diff
@@ -192,6 +197,36 @@ func (e *EnvStackConfig) Parameters() ([]*cloudformation.Parameter, error) {
 	return e.transformParameters(currParams, e.prevParams, transformEnvControllerParameters)
 }
 
+func getSecurityGroupConfig(mft *manifest.Environment) (*template.SecurityGroupConfig, error) {
+	if mft != nil && !mft.Network.VPC.SecurityGroupConfig.IsEmpty() {
+		if !mft.Network.VPC.SecurityGroupConfig.IsValid() {
```

Comment:
shouldn't this be the opposite? also wouldn't validation be handled in the `validate() error` methods in the manifest?

Reply:
I have added this method to `env_validate.go` as you mentioned.